### PR TITLE
chore: prefer properties to basic getter accessors

### PR DESCRIPTION
**Pull request by malandis**
Customers and a language expert have recommended to use properties
where appropriate. Since many response objects just wrap data, we
should use properties to access that data.

For reference: [C# programming guide](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties)

### CODE REVIEW

File: Momento/Responses/CacheGetMultiResponse.cs [hunk 1]
```diff
@@ -5,30 +5,25 @@ namespace MomentoSdk.Responses;
...
     {
-        return responses;
+        get => Responses.Select(response => response.Status).ToList();
     }
 
-    public List<CacheGetStatus> Status()
+    public List<string?> Strings
     {
-        return responses.Select(response => response.Status).ToList();
+	get => Responses.Select(response => response.String()).ToList();
```
Comment (tylerburdsall) [inline Momento/Responses/CacheGetMultiResponse.cs:22]: Maybe this is GitHub rendering it weird, but maybe this needs to be aligned?

File: Momento/Responses/CacheSetMultiResponse.cs [hunk 2]
```diff
@@ -16,13 +16,13 @@ public CacheSetMultiResponse(IDictionary<string, string> items)
         this.items = (object)items;
     }
 
-    public IDictionary<string, string> Strings()
+    public IDictionary<string, string> Strings
     {
-        return (IDictionary<string, string>)items;
+	get => (IDictionary<string, string>)items;
```
Comment (tylerburdsall) [inline Momento/Responses/CacheSetMultiResponse.cs:21]: Same thing

File: Momento/Responses/CacheSetMultiResponse.cs [hunk 3]
```diff
@@ -16,13 +16,13 @@ public CacheSetMultiResponse(IDictionary<string, string> items)
...
     }
 
-    public IDictionary<byte[], byte[]> Bytes()
+    public IDictionary<byte[], byte[]> Bytes
     {
-        return (IDictionary<byte[], byte[]>)items;
+	get => (IDictionary<byte[], byte[]>)items;
```
Comment (tylerburdsall) [inline Momento/Responses/CacheSetMultiResponse.cs:26]: Also here

**Review by tylerburdsall — COMMENTED** [review]
Just some minor formatting comments, everything else looks good

File: Momento/Responses/CacheGetMultiResponse.cs (hunk 1 above)
Comment (kvcache) [reply Momento/Responses/CacheGetMultiResponse.cs:22]: no tabs! Go fix your ide settings 😄

Comment (malandis) [reply Momento/Responses/CacheGetMultiResponse.cs]: Doh! Fixed

File: Momento/Responses/CacheSetMultiResponse.cs (hunk 3 above)
Comment (malandis) [reply Momento/Responses/CacheSetMultiResponse.cs]: Thanks. Fixed.

File: Momento/Responses/CacheSetMultiResponse.cs (hunk 2 above)
Comment (malandis) [reply Momento/Responses/CacheSetMultiResponse.cs]: Thanks. Fixed.

**Review by kvcache — APPROVED** [review]
I like

**Review by cprice404 — APPROVED** [review]
❤️ 😍 ❤️ :ty:

**Review by tylerburdsall — APPROVED** [review]

**Review by eaddingtonwhite — APPROVED** [review]

**Pull request merged**